res/values-de/strings.xml

```diff
@@ -3,35 +3,35 @@
     <string name="app_name">Call recorder for Android</string>
     <string name="menu_settings">Einstellungen</string>
     <string name="menu_about">über</string>
-    <string name="menu_Enable_record">Aktivieren Rekord</string>
-    <string name="menu_Disable_record">Deaktivieren Rekord</string>
+    <string name="menu_Enable_record">Aufnahme aktivieren</string>
+    <string name="menu_Disable_record">Aufnahme deaktivieren</string>
     <string name="title_activity_main">Call recorder for android</string>
-    <string name="list_play_your_recordeed_call">spielen Sie Ihre aufgezeichneten Anruf</string>
-    <string name="list_send_your_recordeed_call">senden Sie Ihre aufgezeichneten Anruf</string>
+    <string name="list_play_your_recordeed_call">spielen Sie Ihre aufgezeichneten Anruf ab</string>
+    <string name="list_send_your_recordeed_call">versenden Sie Ihre aufgezeichneten Anruf</string>
     <string name="list_erase_your_recordeed_call">löschen Sie Ihre aufgezeichneten Anruf</string>
     <string name="receiver_start_call">Aufzeichnungsbeginn Anruf</string>
-    <string name="receiver_end_call">End Record Anruf</string>
+    <string name="receiver_end_call">Stoppe Anrufaufnahme</string>
     <string name="confirm_delete_title">Warnmeldung</string>
-    <string name="confirm_delete_text">Sind Sie sicher, dass Sie diese Aufnahme löschen?</string>
+    <string name="confirm_delete_text">Sind Sie sicher, dass Sie diese Aufnahme löschen möchten?</string>
     <string name="confirm_delete_yes">ja</string>
-    <string name="confirm_delete_no">nicht</string>
+    <string name="confirm_delete_no">nein</string>
     <string name="options_title">Optionen Titel</string>
     <string name="options_delete">Löschen</string>
     <string name="confirm_play">Abspielen</string>
     <string name="confirm_send">Senden</string>
     <string name="sendMail_subject">Ihre Aufzeichnung von Ihrem Telefon</string>
-    <string name="sendMail_body">Dies ist ein Aufruf Rekord von Ihrem Telefon</string>
+    <string name="sendMail_body">Dies ist ein Anrufaufzeichnung von Ihrem Telefon</string>
     <string name="about_title">über</string>
     <string name="about_content">Call recorder by Kobi Krasnoff</string>

```

Comment:
It appears you did not mention Raul Robledo (riul88).

See https://github.com/riul88/call-recorder-for-android/blob/master/res/values/strings.xml#L25